test/domain/model/contact_test.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:fluffychat/domain/model/contact/contact.dart';
+import 'package:fluffychat/domain/model/contact/hash_details_response.dart';
+
+void main() {
+  group('calLookupAddress for email', () {
```

Comment:
i think you should rename file to -> `callLookupAddress_test`

Reply:
no it is `cal` :)

Reply:
nice to have

Reply:
done